doc/glm/meaning_difference.rst

```diff
@@ -0,0 +1,61 @@
+.. _meaning_difference:
+
+==================================================
+Difference in meanings between different toolboxes
+==================================================
+
+.. topic:: **Page summary**
+
+	* SPM has the same meaning of analysis levels, compared to nilearn's models
+		* First-level: analyze across runs
+		* Second-level: group-level analysis
+	* FSL has a different meaning
+		* First-level: analyze one session for one subject
+		* Second-level: analyze across sessions for one subject
+		* Third-level: group-level analysis
+	
+These differences can be put in the table below:
+
++----------------------------------------------+----------------+------------------+---------------+
+| Analyze                                      | nilearn        | SPM              | FSL           |
++==============================================+================+==================+===============+
+| One session, one subject                     | First-level           | First-level             | First\-level  |
++----------------------------------------------+----------------+------------------+---------------+
+| More than one session, one subject           | First\-level   | First\-level     | Second\-level |
++----------------------------------------------+----------------+------------------+---------------+
+| More than or equal to one session,           | Second\-level  | Second\-level    | Third\-level  |
+| more than one subject                        |                |                  |               |
++----------------------------------------------+----------------+------------------+---------------+
+
+*Table showing the differences of the meaning for level of models between toolboxes\/libraries*
+
+Statistical Parametric Mapping (SPM)
+====================================
+
+SPM uses the same notation as nilearn for analysis levels, with a note that a session still refers to an imaging session or a run, and within a run there could be multiple conditions (e.g., congruent and incongruent).
+In this case, `SPM`_ provided `tutorials`_ and documentation, including `lectures`_, which one could learn to analyze their own fMRI data with the meaning of analysis levels being as follows:
+
+	* `First-level analysis in SPM`_: Analyze across sessions for a subject (i.e., more than one session of one subject)
```

Comment:
I suspect that we should mention somewhere that it is typical in the SPM workflow to put all runs in a single design matrix, where as nilearn typically will give you one design matrix per run.

Reply:
Gotcha @Remi-Gau , that's a really great point! However, I'm wondering if I understand the concept correctly, so if there are different conditions, do they still count as a single run if the imaging acquisition is continuous, uninterrupted, and comprise of the same parameters?

I'm looking at [nilearn example](https://nilearn.github.io/dev/auto_examples/04_glm_first_level/plot_design_matrix.html) it seems like there are different conditions within a design matrix, but you said that nilearn gives one design matrix per run, so I'm just not sure if I understand the concept of run correctly as I should or not.

But the [SPM](https://andysbrainbook.readthedocs.io/en/latest/SPM/SPM_Short_Course/SPM_Statistics/SPM_06_Stats_Running_1stLevel_Analysis.html) really does have a design matrix that goes through all the runs in one matrix.

Thank you!!